### ISSUE TITLE
[BUG] Arabic at random does not select randomly

### ISSUE THREAD

**Issue by Felienne** [Felienne/hedy#1867]
```
صديقي هو احمد، خالد، حسن
قول صديقي بشكل عشوائي
```

Level 3 example code prints something but does not seem to print a random thing. I assume this is because the list is not separated by the commas that we support `,` but with `،`?

@alaaeddin-swidan I tried to change the ، into , but since I have no clue what the code says it was hard to get right. Can you try this?

And... should we change the grammar to allow ، as list separator? Would that be most natural or could you use , too?

**Comment by Felienne** [comment]
Haha, this code does work!

```
صديقي هو احمد, خالد, حسن
قول صديقي at random
```

So maybe there is an issue with the list separator and also with parsing the words `at random` in Arabic?

**Comment by alaaeddin-swidan** [comment]
Good catch @Felienne! 
I tried the codes and i think you are right, there are two things here: 
- the comma: I think we need to support the RTL comma. Putting myself in the shoes of a child writing using a RTL keyboard it will be difficult to switch keyboards back and forth just for the comma -- and of course we need then extra instructions to let them know that they need to do this. 
- the at random: i translated this into two words (separated by a space) in Arabic, basically with at being a word (بشكل) and random being another word (عشوائي). I am not sure how you work things out in the commands parser, i assume there is something to change there. Otherwise i could come with a one-word translation and it might then be resolved. 
-

**Comment by Felienne** [comment]
Ah just checked and this was fixed by #1868!
